Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -318,8 +326,14 @@ function Get-MetadataDatabaseConnectionString([string] $metadataDbName, [string]
     }
     $metadataConnStr = "Server=$($sqlServerAddress);Database=$($metadataDbName);Trusted_Connection=True;MultipleActiveResultSets=True;"
 
-    Invoke-Sql $metadataConnStr "SELECT TOP 1 RoleID FROM CatalystAdmin.RoleBASE" | Out-Null
-    Write-DosMessage -Level "Information" -Message "Metadata DB Connection string: $metadataConnStr verified"
+    try {
+        Invoke-Sql $metadataConnStr "SELECT TOP 1 RoleID FROM CatalystAdmin.RoleBASE" | Out-Null
+        Write-DosMessage -Level "Information" -Message "Metadata DB Connection string: $metadataConnStr verified"
+    }
+    catch {
+        Write-Error $_.Exception -ErrorAction Stop
```

Comment:
Same comment as above...

Reply:
We can change this. We are using `Write-Error` in some places. I'll change to use the Dos library function.

Reply:
We use Write-Error in Fabric-Install-Utilities, we shouldn't be using it anywhere else, we want to move to DosInstallUtilities for new stuff.